client/src/App.js

```diff
@@ -43,7 +43,7 @@ function App() {
         {/* <LinearGradient width="100px" />
         <LinearGradient width="300px" />
         <LinearGradient width="500px" /> */}
-        <nav className="fixed w-full z-10 flex items-center justify-between h-16 backdrop-filter backdrop-blur-lg bg-gray-600 bg-opacity-50">
```

Comment:
I think we want to keep the navbar in `fixed` position. Is there a reason we removed it here?

Reply:
I wasn't the one to move it. Maybe I should ask the team?

Reply:
yeah I removed it from nav instead of footer. I added it back.